tests/unit/test_local_engine.py

```diff
@@ -0,0 +1,215 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+import pandas.testing
+import pyarrow as pa
+import pytest
+
+import bigframes
+import bigframes.pandas as bpd
+from tests.system.utils import skip_legacy_pandas
+
+pytest.importorskip("polars")
+
+
+# All tests in this file require polars to be installed to pass.
+@pytest.fixture(scope="module")
+def polars_session():
+    from . import polars_session
+
+    return polars_session.TestSession()
+
+
+@pytest.fixture(scope="module")
+def test_frame() -> pd.DataFrame:
```

Comment:
Could we rename this? The leading `test_` threw me for a loop before I realized it was a fixture.

Reply:
renamed